pages/docs/workflows.md

```diff
@@ -527,3 +527,41 @@ run = 'deno --unstable run -A --no-check --import-map importmap.json deno-genera
 ```
 
 For an easier `chomp deno-generate` execution that will first ensure the import map is up to date in the task graph.
+
+## Vitejs
+
+ES-Modules are supported directly in all major [browsers](https://caniuse.com/es6-module) now. And with [es-modu-shims](https://github.com/guybedford/es-module-shims) you can polyfill the rest of the missing features and ship es-modules with confidence.
+Vitejs is built heavily on the esm modules which makes both jspm and vitejs a perfect combination. 
+
+The dependencies are served from jspm, and the project is bundled using vitejs. We released `vite-plugin-jspm` to enable this workflow. You can download the plugin using.
+
+```sh
+npm install vite-plugin-jspm --save-dev
+```
+
+
+You can add to the config and disable the `polyfilling` and `preload` options from `vitejs` build. As jspm add `es-module-shims` which takes care of polyfilling.
```

Comment:
Perhaps rather start with the next line:

```suggestion
`vite` and `vite build` can be used dev-server and production builds respectively. For the dev build, JSPM will inject ES Module Shims to polyfill import maps, while in the production build all modules will be build from the CDN. The plugin takes all the options that are supported with [@jspm/generator](https://github.com/jspm/generator#options).

For the production build, it can be useful to disable the `polyfilling` and `preload` options:
```

Actually thinking about this though - why aren't these options the default for the build in the JSPM Vite plugin?